### DISCUSSION TITLE
Usage per project using the API

### DISCUSSION THREAD

**Discussion by advany** [vercel/vercel#8310] [Help] [answered]
How can I get the usage statistics using the api? Especially usage statistics per project would be great so I can bill my customers if they generate lots of traffic or call a lot of serverless functions.

**Comment by amyegan** [comment, marked as answer]
You can view [usage](https://vercel.com/docs/concepts/limits/usage) stats at https://vercel.com/dashboard/usage. You can filter by Project to see usage specific to that Project. The API and CLI do not currently return this information.

**Reply by advany** [reply]
@amyegan is there any other way to see around a request of a serverless function what resources are used? So I can keep my own records?

**Reply by amyegan** [reply]
Hi @advany. Sorry for my delayed response (I was out of the office). You could set up [Log Drains](https://vercel.com/blog/log-drains) using one of the available [integrations](https://vercel.com/integrations#logging) to track your serverless functions. That would allow you to generate some of your own metrics and graphs from the logs.
